doc/source/api/api-overview.md

```diff
@@ -8,4 +10,10 @@ For example, the `multimeta` function has a signature of `ce.api.multimeta(sesh,
 
 The query URL `https://base_url/multimeta?ensemble_name=ce_files&model=CanESM2` calls the `multimeta` endpoint and supplies two arguments for the `multimeta` function: `ensemble_name` is "ce_files" and `model` is CanESM2. `sesh` is not supplied in the query URL.
 
-The API function return values are converted to JSON for the endpoint response.
\ No newline at end of file
+The API function return values are converted to JSON for the endpoint response.
+
+## Querying Complex Polygons
+
+Several APIs accept an `area` parameter, which is a WKT string describing a polygon representing the spatial extent of desired data. WKT strings that including many vetices and would result in a URL longer than the standard 4096 bytes may be queried by sending the string in the payload of a `GET` request and including a unique `area_key` parameter in the URL.
+
+It is best to simplify request polygons to the extent possible. The server may time out if asked to return data from a polygon of more than a hundred vertices.
```

Comment:
If I recall correctly, this is what we've historically told users (specifically people uploading their own shapefiles to PCEX), though I think we have improved the back end's efficiency since then.